refiner/app/api/v1/events.py

```diff
@@ -42,15 +40,23 @@ async def get_events(
     """
 
     audit_events = await get_events_by_jd_db(
-        jurisdiction_id=user.jurisdiction_id, db=db, condition_filter=condition_filter
+        jurisdiction_id=user.jurisdiction_id, db=db, cannonical_url=cannonical_url
     )
 
     jd_configurations = await get_configurations_db(
         jurisdiction_id=user.jurisdiction_id, db=db
     )
-    configuration_options = [
-        ConfigurationTrace(name=c.name, id=c.id) for c in jd_configurations
-    ]
+
+    seen_urls = set()
+    configuration_options = []
+
+    for c in jd_configurations:
```

Comment:
I did this using list comprehension initially but mypy was complaining about some return type issues, so did this the generic loop way